core/src/main/java/org/web3j/ens/contracts/generated/PublicResolver.java

```diff
@@ -68,20 +66,17 @@ private PublicResolver(String contractAddress, Web3j web3j, TransactionManager t
         return responses;
     }
 
-    public Observable<AddrChangedEventResponse> addrChangedEventObservable(DefaultBlockParameter startBlock, DefaultBlockParameter endBlock) {
+    public Flowable<AddrChangedEventResponse> addrChangedEventObservable(DefaultBlockParameter startBlock, DefaultBlockParameter endBlock) {
         final Event event = new Event("AddrChanged", 
                 Arrays.<TypeReference<?>>asList(new TypeReference<Bytes32>(true) {}, new TypeReference<Address>() {}));
         EthFilter filter = new EthFilter(startBlock, endBlock, getContractAddress());
         filter.addSingleTopic(EventEncoder.encode(event));
-        return web3j.ethLogObservable(filter).map(new Func1<Log, AddrChangedEventResponse>() {
-            @Override
-            public AddrChangedEventResponse call(Log log) {
-                EventValues eventValues = extractEventParameters(event, log);
-                AddrChangedEventResponse typedResponse = new AddrChangedEventResponse();
-                typedResponse.node = (byte[]) eventValues.getIndexedValues().get(0).getValue();
-                typedResponse.a = (String) eventValues.getNonIndexedValues().get(0).getValue();
-                return typedResponse;
-            }
+        return web3j.ethLogObservable(filter).map(log -> {
```

Comment:
Just realised this is generated code - all generated code must be Java 1.6 compatible

Reply:
Thanks - I didn't know that. Will convert it across.